LINK/usr/bin/manageiq-db-ready

```diff
@@ -0,0 +1,46 @@
+#!/usr/bin/env ruby
+
+require "yaml"
+require "sd_notify"
+require "manageiq-password"
+
+$stdout.sync = true
+
+def wait_for_service(host, port)
+  loop do
+    `ncat #{host} #{port} </dev/null 2>/dev/null`
+    break if $?.success?
+
+    puts "#{host} #{port} - not accepting connections"
+    sleep(10)
+  end
+
+  puts "#{host} #{port} - accepting connections"
+end
+
+def wait_for_database(db_yaml, env = "production")
+  host, port = db_yaml[env].values_at("host", "port")
+  wait_for_service(host, port || 5432)
+end
+
+def wait_for_manageiq_db(db_yaml, env = "production")
+  host, port, dbname, password = db_yaml[env].values_at("host", "port", "database", "password")
+  password = ManageIQ::Password.decrypt(password)
+
+  loop do
+    results = `PGPASSWORD=#{password} psql --host=#{host} --port=#{port || 5432} --dbname=#{dbname} --command="SELECT FROM miq_regions LIMIT 1"`
+    break if $?.success? && results.include?("(1 row)")
+
+    puts "ManageIQ DB not ready yet"
```

Comment:
We might not want the ManageIQ string printed because of productization - I don't mind the filenames, cause most people don't see that, but these strings will be visible.

Reply:
Suggestions?  `DB not ready yet` is too generic

Reply:
Going to bypass all of this and just say that `#{dbname} is not ready yet`